extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderLightbox.php

```diff
@@ -81,36 +82,46 @@
 					</select>
 				</div>
 				<div class="input-group">
-					<label for="occupation"><?= wfMsg('user-identity-box-about-occupation'); ?></label>
-					<input type="text" name="occupation" value="<?= $user['occupation']; ?>" maxlength="<?= $charLimits['occupation']; ?>" />
+					<label for="occupation"><?= wfMessage( 'user-identity-box-about-occupation' )->escaped(); ?></label>
+					<input type="text" id="occupation" name="occupation" value="<?= $user['occupation']; ?>" maxlength="<?= $charLimits['occupation']; ?>"/>
+				</div>
+				<div class="input-group">
+					<label for="gender"><?= wfMessage( 'user-identity-box-about-gender' )->escaped(); ?></label>
+					<input type="text" id="gender" name="gender" value="<?= $user['gender']; ?>" maxlength="<?= $charLimits['gender']; ?>"/>
 				</div>
 				<div class="input-group">
-					<label for="gender"><?= wfMsg('user-identity-box-about-gender'); ?></label>
-					<input type="text" name="gender" value="<?= $user['gender']; ?>" maxlength="<?= $charLimits['gender']; ?>" />
+					<label for="website"><?= wfMessage( 'user-identity-box-about-website' )->escaped(); ?></label>
+					<input type="text" id="website" name="website" value="<?= $user['website']; ?>">
 				</div>
 				<div class="input-group">
-					<label for="website"><?= wfMsg('user-identity-box-about-website'); ?></label>
-					<input type="text" name="website" value="<?= $user['website']; ?>">
+					<label for="twitter"><?= wfMessage( 'user-identity-box-about-tweet' )->escaped(); ?></label>
+					<label for="twitter" class="tweet-at">twitter.com/</label>
+					<input type="text" id="twitter" name="twitter" value="<?= $user['twitter']; ?>">
 				</div>
 				<div class="input-group">
-					<label for="twitter"><?= wfMsg('user-identity-box-about-tweet'); ?></label>
-					<span class="tweet-at">@</span>
-					<input type="text" name="twitter" value="<?= $user['twitter']; ?>">
+					<label for="fbPage"><?= wfMessage( 'user-identity-box-about-fb-page' )->escaped(); ?></label>
+					<label for="fbPage" class="fb-start">facebook.com/</label>
+					<input type="text" id="fbPage" name="fbPage" value="<?= $user['fbPage']; ?>">
```

Comment:
Awesome, thanks for the clarification @Grunny. I hadn't considered @kvas-damian point considering the validity of apostrophe's in non-link attributes, but I'm glad to hear that those cases will still be handled properly. These values aren't only used as attributes, they're sometimes found inside of `<span>` tags, so I think using ` Sanitizer::encodeAttribute` might be slightly misleading but it's good to know for the future.

Damian, it sounds like we'd be fine to add that `ENT_QUOTES` option here and it should make this code more secure. Since we're at the tail end of the sprint and it'd be great if Rebekah could QA this today, I'm going to make that change and move this into QA. If you have any objections or concerns we can certainly revert that change, but I think it makes sense given the circumstances.